seed/product-seeder.js

```diff
@@ -16,31 +16,37 @@ const deleteData = () => {
     });
 };
 
-const seedData = () => {
-  const items = Product.find();
-  if (items.length < 0) {
-    let products = [];
-    for (let i = 0; i < 24; i++) {
-      let item = new Product({
-        imagePath: faker.image.food( 640, 480, true),
-        title: faker.commerce.productName(),
-        description: faker.commerce.productDescription(),
-        price: faker.commerce.price(10, 90, 0),
-      });
-      products.push(item);
-    }
-    console.log("seedData!");
+async function seedData() {
+  try {
+    const query = await Product.find();
+    query.count(function (err, count) {
```

Comment:
You can use .length property here instead, check if it's not empty and there will be no need for query.count in the next line.